topics/ecology/tutorials/Metashrimps_tutorial/tutorial.md

```diff
@@ -0,0 +1,134 @@
+to produce Data Papers drafts and FAIR quality assessment reports from metadata using EML
+standard.
+This tutorial purpose is also to explain why it is needed to improve the overall FAIR quality
+of metadata and how this tool responds to this necessity.
+
+> <comment-title>What does FAIR mean?</comment-title>
+> [FAIR](https://www.go-fair.org/fair-principles/) stand for **Findable, Accessible, Interoperable, Reusable**. 
+>
+>
+><img src="./Images/FAIR_data_principles.jpg" alt="FAIR Data Principles" width="500"/>
+
+>These principles were [officialy instaured in 2016](https://doi.org/10.1038/sdata.2016.18) to improve the access and usabiliy of data by the machine and to help making data reusable and shareable for users.
+>Metadata is the data used to describe and explain all the context behind the production of data. It is necessary to produce a rich and FAIR metadata in order 
+>to permit external users to understand and reuse data for their own studies.
+{:  .comment}
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+# How can this tool improve the metadata quality?
+
+The purpose of this tool is to help the user improve their metadata quality in order to increase its value to the scientific community and to help highlighting
+the work of all the producers of the data.
+To respond to this objective, this tool aims to give an easy access to a quality assessment report of EML metadata which could guide the producers of
+data/metadata to the production of a metadata with high degree of FAIRness.
+The other objective of this tool is to highlight the work of all of the people that helped in producing this data by giving the access of a draft of Data Paper
+that is reuseable either in a non editable HTML file, that can represent the metadata in a more ergonomic way facilitating its understanding and shareability,
+or in an editable docx file. Having an editable Data Paper draft will allow the producer to complete/modify the draft of Data Paper so that it could become
+publishable as a real Data Paper giving recognition to all the people that helped producing the data.
+
+> <comment-title>What is a Data Paper?</comment-title>
+> According to the [GBIF](https://www.gbif.org/data-papers) (Global Biodiversity Information Facility), 
+>A data paper is a peer reviewed document describing a dataset, published in a peer reviewed journal. It takes effort to prepare, curate and describe data. 
+>Data papers provide recognition for this effort by means of a scholarly article.
+{:  .comment}
+# Get data
+
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial
+> 2.Import this metadata file from [Zenodo]({{ https://doi.org/10.5281/zenodo.8130567 }}) to test it
+>     -> `{{ Training Data for "Creating Quality FAIR assessment reports and draft of Data Papers from EML metadata with MetaShRIMPS" }}`):
```

Comment:
```suggestion
>     -> Training Data for "Creating Quality FAIR assessment reports and draft of Data Papers from EML metadata with MetaShRIMPS"):
```

`{{ .. }}` marks a variable in Jekyll, here it will look for a variable with that name and not be able to find it. Please avoid `{{` outside of e.g. page.zenodo_link where you want to access a variable.